lib/normalizers/v1.js

```diff
@@ -42,16 +42,14 @@ _.assign(Builders.prototype, {
     auth: function (entityV1) {
         if (!entityV1) { return; }
         if (util.notLegacy(entityV1, 'auth') && entityV1.auth) {
-            if (entityV1.auth.type === 'noauth') { return; }
-
-            return util.sanitizeAuthArray(entityV1);
+            return (entityV1.auth.type === 'noauth') ? entityV1.auth : util.sanitizeAuthArray(entityV1);
```

Comment:
Can we return `{ type: 'noauth' }`, so any other properties in `auth` are removed?